packages/copy/README.md

```
@@ -0,0 +1,37 @@
+pnpm add -D @mheob/rollup-plugin-copy
+```
+
+## Usage
+
+```js
+// rollup.config.mjs
+import { copy } from '@mheob/rollup-plugin-copy';
+
+export default {
+	input: 'src/index.js',
+	output: {
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)